numpy/core/tests/test_multiarray.py

```diff
@@ -6331,7 +6274,7 @@ def test_matmul_raises(self):
             assert_raises(TypeError, self.matmul, np.void(b'abc'), np.void(b'abc'))
             assert_raises(ValueError, self.matmul, np.arange(10), np.void(b'abc'))
 
-    def test_matmul_inplace():
+    def test_matmul_inplace(self):
```

Comment:
it's not clear to me why I had to add self here but to get tests to pass locally I needed it.

Reply:
It's because you forgot to unindent it